src/main/resources/errors.yaml

```diff
@@ -781,7 +783,7 @@ request-errors:
 
   - scope: SCHEMA
     code: UNKNOWN_VECTOR_METRIC
-    title: Vector index creation with unknown vector metric
+    title: Vector metric is unknown
     body: |-
       The command attempted to create an vector index using a metric that is not known by the API.
```

Comment:
Same as with most messages: should include name of unknown metric as template.
(follow-up pr).